data-browser/src/helpers/setCookie.ts

```diff
@@ -0,0 +1,8 @@
+const ONE_HOUR = 60 * 60 * 1000;
+
+export const setCookie = (name: string, value: string) => {
+  const expiry = new Date(Date.now() + ONE_HOUR).toISOString();
```

Comment:
What happens if the expires date is passed? A new cookie should probably be set somewhere (auth error handler?)

Reply:
The current version is updated whenever the agent change function is called (setAgentIfChanged).